### ISSUE TITLE
Let users specify the language of a piece of site content in TinyMCE

### ISSUE THREAD

**Issue by joemull** [BirkbeckCTP/janeway#4139]
## User story

As a press manager I may need to specify a different language for one piece of content in a larger website. For example, most of my press site may be in English, but a few of my journals publish in French, so I want to set French on the journal description fields. I want this language to be properly marked up in the eventual HTML so that it is [accessible to screen readers](https://github.com/BirkbeckCTP/hourglass/issues/340):

```html
<html lang="en-GB">
  [most of my site]
  <div class="fr">
    Merci beaucoup  
  </div>
</html>
```

While this option can technically be configured now by changing the settings.py, it would be nice to put it in `TINYMCE_DEFAULT_CONFIG` and consider whether and how it should interact with translation and localization in the rest of Janeway.

## Proposed solution
Configure TinyMCE so that language appears as a dropdown in the toolbar.

- [TinyMCE Content Localization](https://www.tiny.cloud/docs/tinymce/latest/content-localization/)

- [Available Toolbar Buttons](https://www.tiny.cloud/docs/tinymce/latest/available-toolbar-buttons/)

**Comment by joemull** [comment]
Refinement:

What should the language options on the toolbar be? Should it be limited to the languages enabled for the journal? Or should it be any language, so that a user can specify an exception for a block of text inside content that is another language?

We might need to see what the standard is for a multilingual journals and translation, from a Dutch or Belgian journal, for example. What are their expectations for the interface? As languages and translation are political, we would want to avoid stepping on toes or taking an unexamined Anglo-centric approach.

Let's do some investigation for this.

**Comment by StephDriver** [comment]
> What should the language options on the toolbar be? Should it be limited to the languages enabled for the journal? Or should it be any language, so that a user can specify an exception for a block of text inside content that is another language?

In my opinion, it needs to be possible to do this for any language, as per #4218, but I also  for ease of use, we could have a shorter list  of 'frequently used languages' that show up as the first options, that could default to the journal languages plus any others set by journal staff via a setting?

I don't think any language should be excluded or banned.... I think that starts getting dangerous.

**Comment by joemull** [comment]
Yeah, no one is "banning" languages here! What made you think that?

I tend to agree with you on allowing the full list of ISO-defined languages that Janeway knows about.

**Comment by StephDriver** [comment]
> Yeah, no one is "banning" languages here! What made you think that?

I didn't think we were... it was an emotive way to show the end point of making it a choice as to which to include... then allows it be a a choice of which to exclude... when then... etc.
